Handle error during dict expansion

ansley opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* **#51374 Handle error during dict expansion**

Differential Revision: [D26155995](https://our.internmc.facebook.com/intern/diff/D26155995)

Commit:
Handle error during dict expansion

[ghstack-poisoned]

gmagogsfm reviewed (approved):
Thanks for fixing it!

Commit:
Update on "Handle error during dict expansion"


Differential Revision: [D26155995](https://our.internmc.facebook.com/intern/diff/D26155995)

[ghstack-poisoned]
